.eslintrc.json

```diff
@@ -15,5 +15,8 @@
   "plugins": ["react", "@typescript-eslint"],
   "rules": {
     "react/prop-types": "off"
+  },
+  "globals": {
+    "JSX": true
```

Comment:
Can we get this removed?

Reply:
I have removed this and also fixed the other bug in `IconSet.tsx` file. Please check.